frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -783,6 +783,12 @@ function FeatureFlagRollout({ readOnly }: { readOnly?: boolean }): JSX.Element {
                                 })
                             }}
                             label="Enabled"
+                            disabled={!featureFlag.can_edit}
+                            disabledReason={
+                                !featureFlag.can_edit
+                                    ? "You have only 'View' access for this feature flag. To make changes, please contact the flag's creator."
```

Comment:
Subjective suggestion for slightly cleaner copy:
```suggestion
                                    ? "You only have view access to this feature flag. To make changes, contact the flag's creator."
```